haiku-apps/wakeup/wakeup-1.0.recipe

```diff
@@ -0,0 +1,45 @@
+
+SUMMARY="An application that plays sound at given interval"
+DESCRIPTION="WakeUp v1.0,\
```

Comment:
Remove "WakeUp v1.0,\  +Copyright (c)1999." and  "Compiled under DeOS R4.5.1 (Intel Only)"

Try to come up with a better description, maybe point out it can be used as an alarm clock or a tea timer etc...